src/web/src/alert_rules.ecpp

```diff
@@ -53,13 +53,14 @@ std::string checked_name;
     }
 
     std::string name = request.getArg ("name");
+    if(request.getMethod () == "PUT")
```

Comment:
Can you please explain a bit why there is no name check when POST is used?

Reply:
This is to be aligned with the RFC-11 specification
It is a common behavior in the REST API to create a new element (likes assets, gpio, ..)

Reply:
Well, I just wanted to make sure there is no hidden hack below that would make problems to POST. If not, LGTM. Polishing might be done later.